src/Tgstation.Server.Host/Components/InstanceManager.cs

```diff
@@ -675,6 +675,11 @@ void CheckSystemCompatibility()
 			{
 				if (!systemIdentity.CanCreateSymlinks)
 					throw new InvalidOperationException($"The user running {Constants.CanonicalPackageName} cannot create symlinks! Please try running as an administrative user!");
+
+				if (!platformIdentifier.IsWindows && systemIdentity.IsSuperUser)
```

Comment:
Sorry to keep coming back to this, but I forgot this will always fire on docker.

Please create an `InternalConfiguration.RunningInDocker` setting and make sure the docker image has it set by adding `ENV Internal__RunningInDocker=true` near the end of `build/Dockerfile`. The warning should not fire if it's `true`.

`InstanceManager` doesn't have a reference to the internal config yet so you'll also have to add one. It's the same as the already present `swarmConfiguration` field.